docker-compose.yml

```diff
@@ -91,26 +107,49 @@ services:
       context: ./client-participation
       args:
         GIT_HASH: "${GIT_HASH}"
-
+    depends_on:
+      - "config"  
+    volumes:
+      - "config:/config"
+ 
   client-admin:
     container_name: polis-client-admin
     image: polisdemo/client-admin:dev
     build:
       context: ./client-admin
       args:
         GIT_HASH: "${GIT_HASH}"
+    depends_on:
+      - "config"        
+    volumes:
+      - "config:/config"
 
   client-report:
     container_name: polis-client-report
     image: polisdemo/client-report:dev
     build:
       context: ./client-report
+      args:
+        GIT_HASH: "${GIT_HASH}" 
+    depends_on:
+      - "config"
+    volumes:
+      - "config:/config"
+
+  config:
+    container_name: polis-config
+    image: polisdemo/config:dev
+    build:
+      context: ./config
       args:
         GIT_HASH: "${GIT_HASH}"
```

Comment:
There's no dockerfile in this build context. Is that intentional? I believe it's the root of the current error

Reply:
I think that `dockerfile: Dockerfile` is not needed, as it is not used with all containers:

```
grep -e er_name -e build -e context -e docker -e ^\s*$ docker-compose.yml

    container_name: polis-server
    env_file: ./server/docker-dev.env
    build:
      context: ./server
      dockerfile: Dockerfile

    container_name: polis-math
    env_file: ./math/docker-dev.env
    build:
      context: ./math

    container_name: polis-postgres
    env_file: ./server/docker-db-dev.env
    build:
      context: ./server
      dockerfile: Dockerfile-db

    container_name: polis-nginx-proxy
    build:
      context: ./file-server
      dockerfile: nginx.Dockerfile

    container_name: polis-file-server
    build:
      context: ./file-server
      dockerfile: Dockerfile

    container_name: polis-client-participation
    build:
      context: ./client-participation

    container_name: polis-client-admin
    build:
      context: ./client-admin

    container_name: polis-client-report
    build:
      context: ./client-report

    container_name: polis-config
    build:
      context: ./config
```

Reply:
Hm. Maybe, but when it's not specified, it's because the default is assumed (`Dockerfile` in root of context dir)